pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -215,9 +241,14 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) reconcile(request reconcile.Requ
 			}
 		}
 
-		err = csi.SaveClusterConfig(r.context.Clientset, buildClusterID(cephFilesystemSubVolumeGroup), cephCluster.Namespace, r.clusterInfo, nil)
-		if err != nil {
-			return reconcile.Result{}, errors.Wrap(err, "failed to save cluster config")
+		// We want this separate check to allow the SaveClusterConfig() call for external clusters, even though not sure it's necessary.
+		if len(cephFsSvgList.Items) <= 1 {
+			err = csi.SaveClusterConfig(r.context.Clientset, buildClusterID(cephFilesystemSubVolumeGroup), cephCluster.Namespace, r.clusterInfo, nil)
+			if err != nil {
+				return reconcile.Result{}, errors.Wrap(err, "failed to save cluster config")
+			}
+		} else {
```

Comment:
This `else` block for the log message would make more sense above on line 242, since it corresponds to skipping the `deleteSubvolumeGroup()`.

Reply:
@travisn if we move above, the log message will not be printed for the external cluster, and since we are keeping the behaviour the same for the external cluster, wdyt?

Reply:
I think the message we already have for external clusters is ok, no need for this new message for external clusters: 
```
external subvolume group %q deletion is not supported, delete it manually
```